### PR TITLE
App Store and Google Play buttons so we can translate them

### PR DESCRIPTION
**Pull request by TheKishPatel**
**Problem**: We need 'download the app' buttons for our mobile website in a language our customers understand so they can get the app if they want it. 

**Another problem**: Google and Apple only provide translated buttons as png images..... 

**Solution?**: Dynamic buttons that looks almost identical!!!!!! 

See below

image

### CODE REVIEW

File: dist/css/bootstrap.css [hunk 1]
```diff
@@ -5768,6 +5768,93 @@ fieldset[disabled] .btn-link:hover,
...
+}
+.btn-download .badge {
+  color: #000;
+  background-color: #fff;
+}
+.btn-download .icon-download {
+  position: absolute;
+  top: 12px;
+  left: 16px;
+  float: left;
```
Comment (KarlEST) [inline dist/css/bootstrap.css]: What's the reason for float when we have position: absolute already?

**Review by KarlEST — APPROVED** [review]
Tiny question otherwise seems okey

**Comment by TheKishPatel** [comment]
@KarlEST You're right, changed

**Comment by TheKishPatel** [comment]
Fixing conflicts and bumping version

**Pull request merged**